packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -81,4 +85,30 @@ router.get(
   })
 );
 
+/**
+ * POST /internal/carequality/directory
+ *
+ * Registers the organization with the Carequality Directory.
+ */
+router.post(
+  "/directory",
```

Comment:
It seems odd to have an endpoint called `/directory` in which a POST to it will create our or on that directory. Likewise the PUT.

RESTful endpoints associate the method/operation with the noun, and in this case we're naming it "directory" but working with "organization".

Maybe rename the endpoint to `/directory/organization`?

Reply:
Im getting conflicting feedback between you and Dima here haha I had it named `/directory/organization`, and he told me to simplify it to just `/directory`